packages/eas-cli/src/commands/update/index.ts

```diff
@@ -526,6 +551,59 @@ export default class UpdatePublish extends EasCommand {
       json: flags.json ?? false,
     };
   }
+
+  public async getBranchNameFromChannelNameAsync(
```

Comment:
prefer to put in utils (separate module) so that it can be tested in isolation from oclif command instantiation

Reply:
I am trying to do this, however, I am running into a lot of "dependency cycle" issues. For instance, moving this method out as a function to update/utils makes this cycle
- **update/utils** requires `ensureBranchExistsAsync()` from **branch/queries**. When we find no channel, then we create a channel and a branch, so I re-use our same branch logic.
- **branch/queries** requires both `formatBranch()` and `getBranchDescription()` from **update/utils**. Those are both functions specific to the `eas update` command, which output CLI info.

There's another dependency cycle with channels. Is the answer here to (1) create new files to avoid cycles (2) refactor a lot of methods to untangle the cycles somehow? (3) do nothing for now and make a task?

Reply:
Yep, adding a new file is fine. Can just call it `update/getBranchNameFromChannelNameAsync.ts`